src/app/qgisapp.cpp

```diff
@@ -2120,6 +2122,7 @@ void QgisApp::createActions()
   connect( mActionSnappingOptions, &QAction::triggered, this, &QgisApp::snappingOptions );
   connect( mActionOffsetCurve, &QAction::triggered, this, &QgisApp::offsetCurve );
   connect( mActionReverseLine, &QAction::triggered, this, &QgisApp::reverseLine );
+  connect( mActionTrimExtendFeature, &QAction::triggered, this, &QgisApp::trimExtendFeature );
```

Comment:
I'd like to see us keep moving towards just using lambda here, instead of adding more single-use slots to the horrendously bloated QgisApp api.

E.g.
    
    connect( mActionTrimExtendFeature, &QAction::triggered, this, [=]{ mMapCanvas->setMapTool( mMapTools.mTrimExtendFeature ); } );